packages/amplify-cli-core/src/help/commands-info.ts

```diff
@@ -0,0 +1,1257 @@
+export const commandsInfo = [
```

Comment:
There's a lot of structure here that is implicit. It should be explicit instead with type checking. Additionally, the keys in these objects should not be in quotes.